docs/content/Getting-Started/user-quickstart-test.md

```diff
@@ -164,16 +164,21 @@ mkdir -p ${PWD}/temp-space-config
            end="<!--kubestellar-apply-syncer-end-->"
          %}
 
-Wait for the mailbox controller to create the corresponding mailbox workspaces and remember them.
+Wait for the mailbox controller to create the corresponding mailbox spaces and remember them.
 
 ```shell
-# TODO: the mailbox workspaces have not been converted into spaces yet.
-KUBECONFIG=ks-core.kubeconfig kubectl ws root
-while [ $(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces | grep -c -e -mb-) -lt 2 ]; do sleep 10; done
-MB1=$(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster1") | .name')
+while [ $(KUBECONFIG=$SM_CONFIG kubectl get spaces -A | grep -c -e -mb-) -lt 2 ]; do sleep 10; done
+MB1=$(KUBECONFIG=$SM_CONFIG kubectl get spaces -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster1") | .name')
 echo The mailbox for ks-edge-cluster1 is $MB1
-MB2=$(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster2") | .name')
+MB2=$(KUBECONFIG=$SM_CONFIG kubectl get spaces -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster2") | .name')
 echo The mailbox for ks-edge-cluster2 is $MB2
+
+MB1_SPACE="${PWD}/temp-space-config/${MB1}"
+kubectl-kubestellar-get-config-for-space --space-name $MB1 --provider-name default --sm-core-config $SM_CONFIG --output $MB1_SPACE
+MB2_SPACE="${PWD}/temp-space-config/${MB2}"
+kubectl-kubestellar-get-config-for-space --space-name $MB2 --provider-name default --sm-core-config $SM_CONFIG --output $MB2_SPACE
+WMW1_SPACE_CONFIG="${PWD}/temp-space-config/spaceprovider-default-wmw1"
```

Comment:
(1) We are renaming "WMW" to "WDS", so putting "SPACE" in this variable name is redundant.

(2) I would remove "temp-space-config/" from this pathname.

(3) I would make the filename end with ".kubeconfig".